voilab-table.js

```diff
@@ -126,23 +145,29 @@ var lodash = require('lodash'),
         });
 
         self.pdf.y = pos.y + row._renderedContent.height;
-
-        self.pdf.moveDown(self.minRowHeight);
     },
 
     setRowHeight = function (self, row, isHeader) {
         var max_height = 0;
 
-        row._renderedContent = {data: {}, dataHeight: {}};
+        row._renderedContent = {data: {}, dataHeight: {}, contentHeight: {}};
 
         lodash.forEach(self.getColumns(), function (column) {
             var renderer = isHeader ? column.headerRenderer : column.renderer,
                 content = renderer ? renderer(self, row, false) : row[column.id],
-                height = !content ? 1 : self.pdf.heightOfString(content, lodash.assign(lodash.clone(column), {
-                    width: column.width - getPaddingValue('horizontal', column.padding)
-                })),
+                height = !content ? 1 : self.pdf.heightOfString(content, column),
```

Comment:
I saw a problem, when horizontal padding creates a new line of text. I think we should preserve the modified width, taking into account the horizontal padding. Below an image of the problem
![image](https://cloud.githubusercontent.com/assets/5629730/23409733/e7d922f0-fdcc-11e6-8c2e-939e0d45e648.png)